src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,54 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("Monica");
+        manufacturer.setCountry("Mexico");
+        Manufacturer manufacturerMonica = manufacturerService.create(manufacturer);
+        System.out.println(manufacturerMonica);
+
+        long manufacturerId = 11L;
+        System.out.println(manufacturerService.get(manufacturerId));
```

Comment:
Why 11? Why not 111? or 222?
```suggestion
        long manufacturerId = manufacturer.getId();
        System.out.println(manufacturerService.get(manufacturerId));
```